coordinator/src/services/translation/prescription/organization.ts

```diff
@@ -18,14 +19,9 @@ export function convertOrganizationAndProviderLicense(
   fhirHealthcareService: fhir.HealthcareService,
   isCancellation: boolean
 ): peoplePlaces.Organization {
-  const hl7V3Organization = convertRepresentedOrganization(
-    fhirOrganization,
-    fhirHealthcareService,
-    fhirBundle,
-    isCancellation
-  )
+  const hl7V3Organization = convertRepresentedOrganization(fhirOrganization, fhirHealthcareService, fhirBundle)
 
-  if (!isCancellation){
+  if (!isCancellation) {
```

Comment:
This is probably a bigger change than we want to do here, but... we don't actually check the value of `isCancellation` until this point, yet it has been passed down through all the levels of code to get here. Given we also pass the bundle which contains the same information (whether or not the message represents a cancellation) it would be nice to remove the redundant flag, and only obtain the value at this point.

Reply:
Good idea. Don't think it's that bit a change, will look at doing it